src/App.js

```diff
@@ -1,21 +1,21 @@
-
-
-import Cart  from "./pages/Cart/Cart";
-import Products from "./pages/Products/Products"
-import  Login  from "./pages/Authentication/Login";
-import Signup  from "./pages/Authentication/Signup";
-import Wishlist  from "./pages/Wishlist/Wishlist";
-import  Landingpage from "./pages/Landing-page/Landingpage";
+import './App.css';
+import { Routes, Route } from "react-router-dom";
+import { Footer, Navbar } from "./components/index";
```

Comment:
You can remove the `/index` from the path as it is the beauty of react which finds the index.js file first if the filename is not given. You can just give the path as `./components/ `and it will work just fine.

Reply:
Thank you for your suggestion . i will change it.